react/features/google-api/actionTypes.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * The type of Redux action which changes Google API state.
+ *
+ * {
+ *     type: SET_GOOGLE_API_STATE
+ * }
+ * @public
+ */
+export const SET_GOOGLE_API_STATE = Symbol('SET_GOOGLE_API_STATE');
```

Comment:
As we will use the Microsoft API only for the calendar integration, I was thinking to implement it just inside calendar-sync feature. And the google calendar API will have its own place in calendar-sync, but it will be re-using this new feature.